package.json

```diff
@@ -18,25 +19,27 @@
         "style/**/*.{css,js,eot,gif,html,jpg,json,png,svg,woff2,ttf}",
         "src/**/*.{ts,tsx}"
     ],
-    "main": "lib/index.js",
-    "types": "lib/index.d.ts",
-    "style": "style/index.css",
     "repository": {
         "type": "git",
         "url": "https://github.com/QuantStack/jupyterlab-suggestions.git"
     },
+    "workspaces": [
+        "packages/*",
```

Comment:
Looks like we don't have this folder tracked at the moment?

Should the extension frontend package be moved to `packages`? So all JS packages are under `packages`? 

This is approach that other monorepos have been taking, for example: https://github.com/jupyterlab/jupyter-chat

Reply:
ok, cool then let's go with this for now!